docs_user/modules/proc_adopting-image-service-with-block-storage-backend.adoc

```diff
@@ -72,13 +68,15 @@ spec:
                   annotations:
                     metallb.universe.tf/address-pool: internalapi
                     metallb.universe.tf/allow-shared-ip: internalapi
-                    metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+                    metallb.universe.tf/loadBalancerIPs: 172.17.0.80 <1>
                 spec:
                   type: LoadBalancer
           networkAttachments:
             - storage
 ----
 +
+<1> If you use IPv6, change the load balancer IP to the load balancer IP in your environment, for example, `metallb.universe.tf/loadBalancerIPs: fd00:bbbb::80`.
```

Comment:
Maybe: 
"If you use IPv6, ensure that you use the IPv6 address of your load balancer here, such as `metallb.universe.tf/loadBalancerIPs: fd00:bbbb::80`."

Although to me this comment could be universal, and maybe a tiny bit self explanatory

Reply:
It is self-referential. I'd leave the note as it is.